app/serializers/api/v3/participant_declaration_serializer.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require "jsonapi/serializer/instrumentation"
+
+module Api
+  module V3
+    class ParticipantDeclarationSerializer
+      include JSONAPI::Serializer
+      include JSONAPI::Serializer::Instrumentation
+
+      set_id :id
+      set_type :'participant-declaration'
```

Comment:
could we also retain the `has_passed` attribute? (might have been missing from docs but is in v1/v2 serialiser), could we also add to docs?